x-pack/plugins/fleet/public/components/package_policy_actions_menu.tsx

```diff
@@ -37,6 +37,7 @@ export const PackagePolicyActionsMenu: React.FunctionComponent<{
   const { getHref } = useLink();
   const authz = useAuthz();
 
+  const agentPolicy = agentPolicies.length > 0 ? agentPolicies[0] : undefined; // TODO: handle multiple agent policies
```

Comment:
I created a follow up to handle these separately: https://github.com/elastic/kibana/issues/186357